external-import/dogesec-ctibutler/docker-compose.yml

```
@@ -0,0 +1,17 @@
+version: '3'
+services:
+  connector-ctibutler:
+    image: opencti/connector-ctibutler:6.6.4
+    environment:
+      - OPENCTI_URL=http://opencti:8080
+      - OPENCTI_TOKEN=CHANGME
+      - CONNECTOR_ID=CHANGEME
+      - CONNECTOR_NAME=CTIBUTLER
+      - CONNECTOR_SCOPE=identity,attack-pattern,course-of-action,intrusion-set,malware,tool,report,vulnerability,campaign,incident,indicator,infrastructure,location,note,threat-actor,tool,software
```

Small modifications on this file to align with other connectors

```suggestion
    image: opencti/connector-dogesec-ctibutler:6.6.4
    environment:
      - OPENCTI_URL=http://opencti:8080
      - OPENCTI_TOKEN=CHANGME
      - CONNECTOR_ID=CHANGEME
      - CONNECTOR_NAME=CTIBUTLER
      - CONNECTOR_SCOPE=ctibutler
```